UKHomeOffice/sgmr-service
# SPCR-384 Implement initial arrival form

### AC / Description of changes
Create an arrival form using the information the backend currently needs. The form should not have to/from inputs (as these are awaiting backend changes). The port input should use the autocomplete component and the file should have validation.

### To Test
- Click on 'Create a new voyage plan'
- Complete the departure form and continue
- You should see a new arrival form
- Click on 'Save and continue'
- >All fields should error and error links should scroll to and focus on inputs 
- Enter an invalid date (32/13/2022) and invalid time (24:00)
- >Errors should read 'You must enter a valid arrival date/time'
- Enter a date in the past
- >Error should read 'You must enter an arrival date in the future'
- Enter an arrival date before the departure date
- >Error should read 'You must enter an arrival date after your departure'. 
- Enter valid details
- Click save and **exit**
- >You should be taken to the dashboard and see the voyage in the draft
- Enter valid details
- Click save and **continue**
- >You should be taken to the V1 vessel form

### Notes

---
* remember to merge to feature branches not master *
